gandalf.js

```diff
@@ -1,134 +1,134 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.